BackendServices/CastleLibrary/EmotionEngine.Emulator/Ps2Float.cs

```diff
@@ -129,7 +129,7 @@ public Ps2Float Add(Ps2Float addend, bool COP1)
                 a = (uint)(a & temp);
             }
 
-            return new Ps2Float(a).DoAdd(new Ps2Float(b), false);
```

Comment:
Yes, this one is my fault, I did many changes on it when you had the PR open, ideally fuse this file off the pr.